scipy/optimize/_differentialevolution.py

```diff
@@ -489,12 +509,28 @@ class DifferentialEvolutionSolver:
         Specify `seed` for repeatable minimizations.
     disp : bool, optional
         Prints the evaluated `func` at every iteration.
-    callback : callable, `callback(xk, convergence=val)`, optional
-        A function to follow the progress of the minimization. ``xk`` is
-        the current value of ``x0``. ``val`` represents the fractional
-        value of the population convergence. When ``val`` is greater than one
-        the function halts. If callback returns `True`, then the minimization
-        is halted (any polishing is still carried out).
+    callback : callable, optional
```

Comment:
Please apply the changes above here, too. I would also be fine removing and referring to the public function.

Reply:
ACK